incubator/neo4j/templates/core-dns.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "neo4j.fullname" . }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "neo4j.name" . }}
+    component: "{{ .Values.core.name}}"
```

Comment:
- Use consistent formatting. Space after `{{` and before `}}`.
- Don't make components configurable.